concepts/booleans/introduction.md

```diff
@@ -1,49 +1,18 @@
 # Introduction
 
-Go supports the three logical operators `&&` (AND), `||` (OR), and `!` (NOT).
+Booleans in Go are represented by the `bool` type, which values can be either `true` or `false`.
```

Comment:
```suggestion
Booleans in Go are represented by the `bool` type.
A `bool` is either `true` or `false`.
```

Split the sentence since there are two distinct statements rather than joining them with a comma.